src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -62,6 +63,8 @@ private class ConfigurationCacheItem
             _parametersCache = new MemoryCache(new MemoryCacheOptions());
         }
 
+        public ICollection<ITelemetryPublisher> TelemetryPublishers { get; init; }
```

Comment:
I don't believe feature manager needs to add or remove publishers, so should be IEnumerable